src/main/java/org/apache/maven/plugins/dependency/utils/CopyUtil.java

```diff
@@ -72,4 +72,19 @@ public void copyArtifactFile(Artifact sourceArtifact, File destination) throws I
         FileUtils.copyFile(source, destination);
         buildContext.refresh(destination);
     }
+
+    /**
+     * Copies a file to a destination.
+     *
+     * @param source the source file to copy
+     * @param destination the destination file
+     * @throws IOException if copy has failed
+     *
+     * @since 3.2.0
+     */
+    public void copyFile(File source, File destination) throws IOException {
+        logger.debug("Copying file '{}' to {}", source, destination);
+        FileUtils.copyFile(source, destination);
+        buildContext.refresh(destination);
+    }
```

Comment:
I am not sure this is a  good method name: It does not just copy a file, it adds it  to the buildContext. Thus, the method name should reflect this. Otherwise someone could think it dies the same as `FileUtils.copyFile()`.

Reply:
> I am not sure this is a good method name: It does not just copy a file, it adds it to the buildContext. Thus, the method name should reflect this. Otherwise someone could think it dies the same as `FileUtils.copyFile()`.

@bmarwell do you think something like `copyFileAndRefreshContext`is good?

Reply:
I would not change the name of the method, because of the symmetric in the class. Take a look at the method `copyArtifactFile`. It also does more than only copy. But I see your point @bmarwell. Maybe we should change the description of the whole classes because the buildcontext is afield of the class.